Prace_domowe/Praca_domowa1/Grupa1/Adam_Frej/PD1.ipynb

```diff
@@ -0,0 +1,1134 @@
+{
```

Comment:
<p>Brakuje komentarza, że w przypadku zmiennej <em>area </em>odchylenie standardowe jest większe od średniej, co jest spowodowane bardzo dużą wartością maksymalną. W takich sytuacjach zmienną zazwyczaj poddaje się logarytmowaniu - ale to nie było wymagane w tej pracy domowej.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>